src/main/java/io/zipcoder/interfaces/Educator.java

```diff
@@ -0,0 +1,44 @@
+package io.zipcoder.interfaces;
+
+public enum Educator implements Teacher{
+    TARIQHOOK (0, "Tariq Hook"),
+    IYASUWATTS (1, "Iyasu Watts"),
+    DOLIODURANT (2, "L. Dolio Durant"),
+    LEONHUNTER (3, "Leon Hunter"),
+    FROILANMIRANDA (4, "Froilan Miranda");
+
+    private final Instructor instructor;
+
+    private double timeWorked =0;
+
+    Educator(int id, String name){
```

Comment:
Perhaps consider using an underscore as a delimiter in the constant declaration.
```
TARIQ_HOOK,
IYASU_WATTS,
DOLIO_DURANT,
LEON_HUNTER,
FROILAN_MIRANDA;
```

This will allow the string representation of the name to be derived via the built in enum method `name()`.
The goal would be to eliminate redundant construction declaration; e.g. - `LEONHUNTER("Leon Hunter")`
```
Educator(int id) {
    this.instructor = new Instructor(id, name());
}
```

Additionally, the `id` could be derived via the built in enum method `ordinal()` method.
```

Educator() {
    this.instructor = new Instructor(ordinal(), name());
}
```